src/libcore/panicking.rs

```diff
@@ -7,7 +7,7 @@
 //!
 //! ```
 //! # use std::fmt;
-//! fn panic_impl(fmt: fmt::Arguments, file_line_col: &(&'static str, u32, u32)) -> !
+//! fn panic_impl(pi: &PanicInfo<'_>) -> !
```

Comment:
Cc https://github.com/rust-lang/rust/issues/66689 -- this is a minimal fix, but I am not sure if the comment further down is still accurate.